content/administrators/people/student-profile.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Student Profile"
+linktitle: "Student Profile"
+---
+
+The student profile is a key component of Gibbon, as it gives administrators and teachers full access to all the necessary information about an individual student.
+
+## General information
+
+The General Information panel lives at the top of a student profile, and provides key information about a student, such as their name, form group, age and email, at a glance. The field types in this area are fixed in the data structures and cannot be modified.
+
+## Teachers of student
+
+Below General Information is an overview of all the student's teachers, based on their timetable. This includes teachers, form tutors, educational assistants, and heads of year. Each entry has the teacher's photo, their name, and what subject they teach the student. If the same teacher teaches one student multiple subjects, they will appear twice.
```

Comment:
There is an option to select a list view of teachers, and in the list view the teacher's names and email addresses can easily be copy-pasted to send emails to a student's teachers.